config.py

```diff
@@ -0,0 +1 @@
+SQLALCHEMY_DATABASE_URI = 'sqlite:////tmp/test.db'
```

Comment:
I've used sqlite as default for now as we don't have setup for "production" environment.
This will create test.db file in /tmp foulder, which, I think, will not work on Windows. Windows users should either redefine this option in custom configuration file or we can just setup Vagrant so everybody will have the same environment.